Classes/common/HTTP/CDTHTTPInterceptorContext.m

```diff
@@ -16,6 +16,12 @@
 
 #import "CDTHTTPInterceptorContext.h"
 
+@interface CDTHTTPInterceptorContext ()
+
+@property (nullable, readwrite, nonatomic, strong) NSURLResponse *response;
```

Comment:
I'm pretty sure it needs to be redeclared in order to get the access to the method to set the iVar.